app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,6 @@
+class DashboardController < ApplicationController
+
```

Comment:
I think we'll want to add some sort of before_action check valid user so not anyone can access dashboard paths